common/src/main/java/org/apache/rocketmq/common/UtilAll.java

```diff
@@ -683,7 +683,7 @@ public static List<String> split(String str, String splitter) {
         }
 
         if (StringUtils.isBlank(str)) {
-            return Collections.EMPTY_LIST;
+            return new ArrayList<>();
```

Comment:
`if (str == null) ` condition can be removed

Reply:
ok